budgeteer-web-interface/src/main/resources/db/migration/V1_1_0__Baseline.sql

```diff
@@ -0,0 +1,856 @@
+--------------------------------------------------------
+--  Datei erstellt -Mittwoch-Dezember-19-2018
+--------------------------------------------------------
+--------------------------------------------------------
+--  DDL for Sequence HIBERNATE_SEQUENCE
+--------------------------------------------------------
+
+   CREATE SEQUENCE  "BUDGETEER"."HIBERNATE_SEQUENCE"  MINVALUE 1 MAXVALUE 9999999999999999999999999999 INCREMENT BY 1 START WITH 72 CACHE 20 NOORDER  NOCYCLE ;
```

Comment:
Why does the sequence start with 72? They all start with 1 in the auto-generated DDL. Maybe you already had some data in the db?

Reply:
That's a good question. I only exportet the sql without data fom oracle database.

Reply:
So i changed the `start with` declaration in baseline to one and it seems to work.

But i saw that the project doesn't work with database version 1.1.0, because there were merged changes that need the `CONTRACT_SORTING` table from version 1.1.1. commit #303 

Circle ci runs the gradle tasks which uses the in memory database, thats the reason why problems with oracle will not be found.

Reply:
Yes, as we develop on master, it is not on 1.1.0, that version can be found on the release branch. Still I think this should be merged into master.